orders/test/conftest.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from orders.models import DeclarativeBase
+
+
+@pytest.fixture(scope='session')
+def db_url():
+    """Overriding db_url fixture from nameko_sqlalchemy"""
```

Comment:
We probably also need to explain somewhere where the `db_session` etc fixtures come from, and why these overrides are necessary.